apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowFilters.cs

```diff
@@ -0,0 +1,407 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Protobuf;
+using System;
+
+namespace Google.Cloud.Bigtable.V2
+{
+    /// <summary>
+    /// Static class with convenience methods for creating various kinds of <see cref="RowFilter"/> instances.
+    /// </summary>
+    public static class RowFilters
+    {
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which applies the given
+        /// label to all cells in the output row. This allows the client to determine
+        /// which results were produced from which part of the filter.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Values must be at most 15 characters in length, and match the RE2
+        /// pattern `[a-z0-9\\-]+`
+        /// </para>
+        /// <para>
+        /// Due to a technical limitation, it is not currently possible to apply
+        /// multiple labels to a cell. As a result, a Chain may have no more than
+        /// one sub-filter which contains an ApplyLabelTransformer. It is okay for
+        /// an Interleave to contain multiple ApplyLabelTransformer filters, as they
+        /// will be applied to separate copies of the input. This may be relaxed in
+        /// the future.
+        /// </para>
+        /// </remarks>
+        /// <param name="label">
+        /// The label to apply to all cells in the output row.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter ApplyLabelTransformer(string label)
```

Comment:
There are some Filters that shouldn't be used by most users, specifically, Label and Sink.  They were used for an obscure HBase filter. I think that we should remove those filters from this helper class.

Reply:
Done